src/theme.js

```diff
@@ -0,0 +1,452 @@
+function getTheme({ colors, name }) {
+  return {
+    name: name,
+    colors: {
+      focusBorder: colors.blue[4],
```

Comment:
I had written a script to reverse-engineer the color reference from the underlying hardcoded hex string. Also, I have verified that the output theme is exactly the same as before.